### DISCUSSION TITLE
Unable to use damask.Grid.load_DREAM3D method

### DISCUSSION THREAD

**Discussion by Zak-ship-it** [damask-multiphysics/DAMASK#24] [Q&A] [answered]
Hello,
I am using DAMASK (version 3.0.0a7.post0), and I need to create .vti/.yaml from DREAM3D files.
However, I have encountered an issue while attempting to load a .dream3d file using the damask.Grid.load_DREAM3D method.
The .dream3d file was generated with the DREAM3D software, and I have attached it to this message for your reference. When I attempt to execute the command `damask.Grid.load_DREAM3D('Cylinder_Synthetic.dream3d')` , I receive the following error message:
`KeyError: "Unable to open object (object 'Phases' doesn't exist)" `
Despite specifying `phases='Phases (partial)'` as indicated by the data I observed when opening the XDMF file. 
I would greatly appreciate any assistance or guidance you could provide to resolve this issue. Thank you in advance.
Regards,
Ali
[cylinder.zip](https://github.com/user-attachments/files/16512265/cylinder.zip)

**Comment by MarDiehl** [comment, marked as answer]
unfortunately, the flexibility of the DREAM.3D format makes it very hard to parse. In your case, there are two groups (folders) that contain the spacing information: `DataContainers/Sampled_Cylinder` and `DataContainers/SyntheticVolumeDataContainer`. The latter one contains all the relevant information, but the first one is used as the base group. Specifying the correct one solves the issue:

```python
g = damask.GeomGrid.load_DREAM3D("Cylinder_Synthetic.dream3d",base_group='DataContainers/SyntheticVolumeDataContainer')
```
